core/client/fs/src/main/java/alluxio/client/file/cache/cuckoofilter/SizeEncoder.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.cache.cuckoofilter;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * A size encoder that supports encode/decode size information.
+ */
+public class SizeEncoder {
```

Comment:
why it's called size encoder?  Looks like it's just an encoder for integers.

Reply:
Because we use this to encode the size information stored in ClockCuckooFilter.